fedora-ci/jobs.md

```diff
@@ -16,4 +16,6 @@ in [Testing Farm](https://testing-farm.io) run with a successfull scratch build.
 
 ### Custom test plans
 
-TBD (The work is planned.)
+Custom [TMT](https://tmt.readthedocs.io/) test plans in [Testing Farm](https://testing-farm.io)
+run with a successfull scratch build. This requires `.fmf/version` to be present and at least one
```

Comment:
```suggestion
run with a successful scratch build. This requires a tmt setup (`.fmf/version` presence used as an indicator) in the dist-git repository and at least one
```

Reply:
I'm not sure `tmt setup` is an established term, what about `TMT metadata` instead?